runtime/dart_vm.cc

```diff
@@ -29,6 +29,10 @@
 #include "third_party/tonic/logging/dart_error.h"
 #include "third_party/tonic/typed_data/typed_list.h"
 
+#ifdef FML_OS_ANDROID
```

Comment:
Van the header be safe to include on any platform? The RequestAffinity call can just return false unconditionally on unsupported platforms.

Reply:
I tried that but #ifdef FML_OS_ANDROID seems to always be false in fml, outside of the platform directory?

Reply:
You need to include `//flutter/fml/build_config.h` to get those definitions.

Reply:
Interesting, will give that a shot!

Reply:
Done, seems to work. big duh on my part.